api/schema.graphql

```diff
@@ -3,27 +3,21 @@ type DailyTotal {
   total: Int
 }
 
-type Mutation {
-  """
-  Flag a phone number
-  """
-  flagPhoneNumber(
-    """
-    the suspects phone number
-    """
-    phoneNumber: String
-  ): PhoneNumberFlaggingsSummary
+type identifierFlaggingsSummary {
+  identifier: String
+  summary: [DailyTotal]
 }
 
-type PhoneNumberFlaggingsSummary {
-  phoneNumber: String
-  summary: [DailyTotal]
+type Mutation {
+  """Flag an identifier"""
+  flagIdentifier(
+    """the suspects identifier (phone no, url or email)"""
+    identifier: String
+  ): identifierFlaggingsSummary
 }
 
 type Query {
-  """
-  Stats about report collection
-  """
+  """Stats about report collection"""
```

Comment:
Just curious, did prettier decide to change the formatting here, or are you still ignoring the file on commits?